soc/xlnx/mbv32/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+zephyr_sources(
+    ${ZEPHYR_BASE}/soc/common/riscv-privileged/soc_irq.S
+    ${ZEPHYR_BASE}/soc/common/riscv-privileged/vector.S
```

Comment:
Just curious, is this the way to do this? litex seems to do this way but no other does.

Reply:
Why do you need to reference `soc_irq.S` from `${ZEPHYR_BASE}`? There are a few lines with `${ZEPHYR_BASE}soc/` in Zephyr codebase. I'm just curious why you are doing this way.  It's more of a question to you than anything else.